src/main/java/com/devcycle/sdk/server/local/DVCLocalClient.java

```diff
@@ -87,6 +85,10 @@ public <T> Variable<T> variable(User user, String key, T defaultValue) {
 
     addDefaults(user);
 
+    if (!configManager.isConfigInitialized()) {
+      System.out.println("Variable called before DVCClient has initialized, returning default value");
```

Comment:
not sure if we should be using an actual logger here? like passing one in to the DVCLocalClient

Reply:
Yeah, we currently don't support third party loggers and are only using `System.out.println` or `System.out.printf`.. it'll come in a future PR :)